hack/runtime-migrator/README-rb.md

```diff
@@ -1 +1,140 @@
-# Runtime Backuper
\ No newline at end of file
+# Runtime Backup and Switch
+
+The `runtime-backup-and-switch` application has the following tasks:
+1. Connect to a Gardener project and KCP cluster.
+2. Retrieve all existing shoot specifications.
+3. For each runtime on the input list that was created by the provisioner (shoot is labelled with `kcp.provisioner.kyma-project.io/runtime-id`):
+   1. Get the `Shoot`, `ClusterRoleBinding`, and `OpenIDConnect` resources.
+   2. Save the backup on a disk.
+   3. Mark the ClusterRoleBindings that were created by the Provisioner with the `kyma-project.io/deprecation` label.
+   4. To make sure KIM controls the runtime, set the `kyma-project.io/controlled-by-provisioner` label to `false`.
+
+## Build
+
+To build the `runtime-backup-and-switch` application, run:
+
+```bash
+go build -o ./bin/runtime-backup-and-switch ./cmd/backup-and-switch
+``` 
+
+## Usage
+
+### Dry Run
+This execution does the following:
```

Comment:
```suggestion
This execution example does the following:
```
I'm not sure if you dropped this word by purpose.

Reply:
It wasn't on purpose. Thanks for noticing!